### ISSUE TITLE
Removing Hash from *.module.css

### ISSUE THREAD

**Issue by tomrule007** [dferber90/jest-transform-css#6]
I was wondering if it is possible to remove hashes from module css classes? 

Like support for the [localidentname](https://webpack.js.org/loaders/css-loader/#localidentname) property option in the normal webpack config.
```js
// webpack.config.js
// ...
{
        test: /\.css$/,
        use: [
          'style-loader',
          {
            loader: 'css-loader',
            options: {
              importLoaders: 1,
              modules: { localIdentName: '[path][name]__[local]--[hash:base64:5]'},
            },
          },
        ],
        include: /\.module\.css$/,
      },
```

Thank you very much for sharing this package. It is extremely helpful removing the hash would just allow me to not have to update my jest screenshots as often when making minor untested css changes.

**Comment by dferber90** [comment]
Damn, what a glorious beard Tom! :)

Ohhh, I guess you're talking about Jest Snapshots, not Jest Screenshots. That sent me in the wrong direction initially 🤦 😄 

Seems like past-me has planned for this feature request, but not implemented it:

https://github.com/dferber90/jest-transform-css/blob/master/postcss-runner.js#L37-L41

Maybe you could open a PR which uses `options.generateScopedName || "[path][local]-[hash:base64:10]"`?

I'm not exactly sure if that's enough or if you'd need to set it in a config file as well (and in which config file, maybe `postcss.config.js`). It's been a while since I worked with that package. Hope this helps point you in the right direction at least!

**Comment by tomrule007** [comment]
Thanks for the speedy response and yes I did mean snapshot. I'll take a look at it tomorrow and see if i can put a pr together.

**Comment by dferber90** [comment]
I just published these changes in v2.1.0. Thanks for your contribution :)
